numpy/core/code_generators/generate_umath.py

```diff
@@ -891,24 +892,25 @@ def make_arrays(funcdict):
             thedict = chartotype1  # one input and one output
 
         for t in uf.type_descriptions:
-            if t.func_data not in (None, FullTypeDescr) and not isinstance(t.func_data, FuncNameSuffix):
+            if (t.func_data not in (None, FullTypeDescr) and
+                    not isinstance(t.func_data, FuncNameSuffix)):
                 funclist.append('NULL')
                 astype = ''
                 if not t.astype is None:
                     astype = '_As_%s' % thedict[t.astype]
-                astr = '%s_functions[%d] = PyUFunc_%s%s;' % \
-                       (name, k, thedict[t.type], astype)
+                astr = ('%s_functions[%d] = PyUFunc_%s%s;' %
+                           (name, k, thedict[t.type], astype))
```

Comment:
Because `%s_functions[%d] = PyUFunc_%s%s;' % (name, k, thedict[t.type], astype)` is just one statement broken at `%`, which is sort of like a binary operator. I don't feel strongly about it, and could easily go either way as there seems to be nothing in PEP8 about that specific case.